packages/tradershub/src/features/cfd/screens/CreatePassword/CreatePassword.tsx

```diff
@@ -28,16 +28,21 @@ const CreatePassword: FC<TProps> = ({ icon, isLoading, onPasswordChange, onPrima
 
     const title = PlatformDetails[platform].title;
```

Comment:
```suggestion
    const title = PlatformDetails[platform].title;
```
const { title } = PlatformDetails[platform]